test/bin/helpers/data/package/valid/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We'll expand the level of these tests when we are sure we actually want the behaviour outlined in this pr